.pfnci/run_test.bat

```diff
@@ -0,0 +1,30 @@
+@echo off
+
+set CUDA_VER=%1
+set PY_VER=%2
+set GPU_LIMIT=%3
+set CUDA_PATH=CUDA_PATH_V%CUDA_VER%
+set PY_PATH=C:\Development\Python\Python%PY_VER%
+set PATH=%CUDA_PATH%\bin;%CUDA_PATH%\libnvvp;%PY_PATH%;%PY_PATH%\Scripts\%PATH%
+
+set PYTEST_ATTR=not slow and not ideep
+
+curl -o xpytest.exe -L https://github.com/chainer/xpytest/releases/download/v0.1.0/xpytest-windows.exe
+
+if %GPU_LIMIT% gtr 0 (
+    rem cannot set variable within if block, so use subroutine
+    cmd /c .pfnci\install_cupy.bat
+) else (
+    set PYTEST_ATTR=not gpu and not cudnn and %PYTEST_ATTR%
+)
+
+rem protobuf==3.7.1 with python3.7 causes "DeprecationWarning"
+rem to avoid the warning, install specific version
+rem see https://github.com/chainer/chainer/issues/7523
+pip install protobuf==3.6.1
```

Comment:
The workaround could be removed because #7529 is merged and a newer `protobuf` is available.